content/4-component-composition/5-context/vue3/UserProfile.vue

```diff
@@ -0,0 +1,15 @@
+<script setup>
+import { inject } from "vue";
+const { user, updateUsername } = inject("userContextProvider");
```

Comment:
To match Svelte example, use context name `user` rather than `userContextProvider`.